### ISSUE TITLE
[A11y] Round 2 Homepage Alerts

### ISSUE THREAD

**Issue by azle** [OregonDigital/OD2#2837]
### Operating System

MacOS

### Browser

Safari

### Affected page

Homepage

### Admin user?

No

### Issue Description

Accessibility Alert errors for the homepage see comments.

### Suggested Fix

See recommended in comments

### Can anybody QA or does this need an expert?

A11y familiarity recommended

### QA Steps

Run wave for accessibility

**Comment by azle** [comment]
Issue #1

Redundant Link
<img width="940" alt="Screen Shot 2023-03-13 at 9 05 06 AM" src="https://user-images.githubusercontent.com/8353860/224759083-e8add187-d889-4f2c-bd10-0aca5d604aa7.png">

Recommended Fix
If possible, combine the redundant links into one link and remove any redundant text or alternative text (for example, if a product image and product name are in the same link, the image can usually be given alt="").

Reference for standards and Guidelines:
[2.4.4 Link Purpose (In Context) (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.4)

[Icon index](https://wave.webaim.org/report#)

**Comment by azle** [comment]
Issue #2

Suspicious link text
<img width="826" alt="Screen Shot 2023-03-13 at 9 07 51 AM" src="https://user-images.githubusercontent.com/8353860/224759773-3be1042a-0552-4351-abd0-24c949e8a68d.png">

Recommended Fix
Where appropriate, reword the link text so that it is more descriptive of its destination when read out of context. Remove any extraneous text (such as "click here").

Reference for standards and guidelines:
[2.4.4 Link Purpose (In Context) (Level A)](https://webaim.org/standards/wcag/checklist#sc2.4.4)

**Comment by azle** [comment]
Issue #3

Decorative Images with hidden aria label or no alt text.

Recommended Fix
alt tags need a space between the quotations marks for screen readers to read as decorative text

Doesn't pass accessibility
`
<img src="img.jpg" alt="">
`

Adding the space between the quotes passes as decorative image
`
<img src="img.jpg" alt=" ">
`

**Comment by azle** [comment]
Issue #4

For somer reason the validator is seeing a hidden search, see screen snap:
<img width="757" alt="Screen Shot 2023-03-13 at 9 24 59 AM" src="https://user-images.githubusercontent.com/8353860/224764127-1fdaf33e-fe70-4bd4-a70c-eb8caac44ca0.png">

Below is the recommended fix but is this something that is needed or can it be deleted?

Recommended Fix:
Ensure the search landmark is implemented properly and surrounds the search area of the page.

**Comment by azle** [comment]
Update: 12/12/2024
No Errors but 16 warnings on https://wave.webaim.org/

<img width="344" alt="Screenshot 2024-12-12 at 1 50 28 PM" src="https://github.com/user-attachments/assets/0d12e3a8-5de2-4942-ae22-760f23102dc0" />
